CMakeLists.txt

```diff
@@ -44,11 +44,11 @@ set_target_properties(${IPFS_API_LIBNAME} PROPERTIES
   VERSION ${PROJECT_VERSION}
 )
 target_link_libraries(${IPFS_API_LIBNAME} ${CURL_LIBRARIES})
-
-install(TARGETS ${IPFS_API_LIBNAME} DESTINATION lib)
-install(FILES include/ipfs/client.h DESTINATION include/ipfs)
-install(FILES include/ipfs/http/transport.h DESTINATION include/ipfs/http)
-
+if(ENABLE_INSTALL)
```

Comment:
That is still disabling the install by default. What about `if(NOT DISABLE_INSTALL)`? Then the default behavior is unchanged and you need to explicitly define `DISABLE_INSTALL` to skip installing.

Reply:
Let me fix it.
Thanks